setup.py

```diff
@@ -31,6 +31,8 @@
         "matplotlib==2.2.2",
         "seaborn==0.8.1",
         "sklearn==0.0",
+        "ipython==6.2.1",
+        "jupyter==1.0.0",
```

Comment:
these shouldn't be requirements, simply add them to `requirements.txt`

Reply:
:ok_woman:

Reply:
94d67f7